### ISSUE TITLE
old show_admin_menu vs e107::getNav()->admin  -  they are not compatible

### ISSUE THREAD

**Issue by Jimmi08** [e107inc/e107#3139]
Old show_admin_menu was replaced by e107::getNav()->admin() in boot.php. It's deprecated now.
But they are not compatible.

I am not able to replace
`show_admin_menu(CONTENT_ADMIN_MENU_LAN_6.": ".$content_heading."", $act, $var, TRUE);`

```
if (!function_exists('show_admin_menu'))
{
	function show_admin_menu($title, $active_page, $e107_vars, $js = FALSE, $sub_link = FALSE, $sortlist = FALSE)
	{
		
		return e107::getNav()->admin($title, $active_page, $e107_vars, false, false, $sortlist);
	}
}
```
because $js parameter is not used anymore.

Result is that instead:
`<a style="cursor:hand; cursor:pointer; text-decoration:none;" onclick="showhideit('catpages');">Category&nbsp;pages</a>`

it returns:
`<a class="link" href="#catpages">&nbsp;Category&nbsp;pages</a>`

Probably this should be replaced by e-expandit, but there is not example how to do it. And because $sub_link is always set as false, it will never work.

**Comment by SimSync** [comment]
@Jimmi08 I'm not sure what exactly you want to do/achieve?
Do you want to use it on a front page?
Do you want to attach js to a menu item?

**Comment by Jimmi08** [comment]
I want/need to use content plugin. It has generated admin menu for each category and menu options are displayed by anchors (similar way as List New options)   (one long page, only one part is displayed, you click on second part, this is hidden and next one is displayed - but not horizontal tabs)

I can't use e107::getNav()->admin() because result is stripped and javascript doesn't work. [first post]

I tried to use solution from List New plugin, but I got error in javascript -  I don't know how to use names (for IDs) to get it work. [second post]

For now -  I replaced new show_admin_menu() old one from time before replacing return e107::getNav()->admin() and everything works.   

But I don't want to change core files.  So how to set menu to display selected part on one page (like accordion or vertical tabs).
